maxim/tests/test_logger_azure_openai.py

```
@@ -8,35 +8,46 @@
-baseUrl = data[env]["baseUrl"]
-repoId = data[env]["repoId"]
+openaiKey = os.getenv("OPENAI_API_KEY")
+apiKey = os.getenv("MAXIM_API_KEY")
+baseUrl = os.getenv("MAXIM_BASE_URL")
+repoId = os.getenv("MAXIM_LOG_REPO_ID")
+azureApiKey = os.getenv("AZURE_OPENAI_API_KEY")
+azureEndpoint = os.getenv("AZURE_OPENAI_ENDPOINT")
 
 logging.basicConfig(level=logging.INFO)
 
 class TestLoggingUsingAzureOpenAI(unittest.TestCase):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)